SECURITY.md

```diff
@@ -0,0 +1,39 @@
+# Security Notice
+
+This is the security notice for all DfE (Department for Education) repositories. The notice explains how vulnerabilities should be reported to the DfE. At the DfE there are cyber security and information assurance teams, as well as security-conscious people within the programmes, that assess and triage all reported vulnerabilities.
+
+## Reporting a Vulnerability
+
+The Department for Education is an advocate of responsible vulnerability disclosure. If you’ve found a vulnerability, we would like to know so we can fix it.
+
+Send details to [admin@digital.education.gov.uk](mailto:admin@digital.education.gov.uk), including:
+
+* the website, page or repository where the vulnerability can be observed
+* a brief description of the vulnerability
+* optionally, the type of vulnerability and any related [OWASP category](https://www.owasp.org/index.php/Category:OWASP_Top_Ten_2017_Project)
```

Comment:
Perhaps we could update the "OWASP category" link to the latest (2021) equivalent of the top 10? 👍 

```
https://owasp.org/Top10/
```

Reply:
Good call, thanks @RogerHowellDfE.

Reply:
I've updated it so it points at the current version of the full list rather than the top ten.